cdap-spark-core-base/src/main/java/io/cdap/cdap/app/runtime/spark/SparkRuntimeUtils.java

```diff
@@ -292,15 +295,15 @@ public static Closeable addEventLoggingListener(SparkRuntimeContext runtimeConte
     File eventLogDir = Files.createTempDirectory("spark-events").toFile();
     LOG.debug("Spark events log directory for {} is {}", programRunId, eventLogDir);
 
-    // EvevntLogginginListener is Scala package private[spark] class.
+    // EvevntLoggingListener is Scala package private[spark] class.
```

Comment:
nit: EventLoggingListener instead of EvevntLoggingListener

Reply:
fixed